pkg/utils/sysinfo_test.go

```diff
@@ -0,0 +1,41 @@
+package utils_test
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/ansible/receptor/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSysCPUCount(t *testing.T) {
+	got := utils.GetSysCPUCount()
+	assert.Positive(t, got)
+
+	commandOutput, err1 := exec.Command("nproc").CombinedOutput()
+
+	assert.Nilf(t, err1, "Unable to get nproc output, output = %s, err = %s", string(commandOutput), err1)
+
+	commandOutputWithout := strings.TrimSpace(string(commandOutput))
+	want, err2 := strconv.Atoi(commandOutputWithout)
+	assert.Nilf(t, err2, "Unable to convert nproc output %s, err = %s", string(commandOutput), err2)
```

Comment:
Great work on getting info from `nproc` and `/proc/meminfo`, really cleaver. 
I would argue that the two `asserts` that check to ensure data has been returned are really asserting vars outside of the receptor code base and arent needed. 

The positive test and the final equal test are both testing the receptor code and I think these comparisons alone should be enough

Reply:
Asserts are now removed